modules/php/Models/Player.php

```diff
@@ -518,4 +518,27 @@ public function react($ids, $ctx)
       }
     }
   }
+
+  public function prepareSelection($card, $selectionCards, $playerIds)
+  {
+    $src = $card->getName();
+    $atom = [
+      'state' => ST_SELECT_CARD,
+//      'type' => 'selection_draw' // TODO: Do we actually need 'type'?
+      'msgActive' => clienttranslate('${you} should choose a card to draw because of ${src_name}'),
+//      'msgWaiting' => clienttranslate('${actplayer} has to react to ${src_name}. You may already select your reaction'),
+      'msgInactive' => clienttranslate('${actplayer} should choose a card to draw because of ${src_name}'),
+      'src_name' => $src,
+      'players' => $playerIds,
+      'isPrivate' => false,
```

Comment:
The function should probably take a boolean argument to specifiy whether the selection is private or not

Reply:
Done